octobot_trading/api/exchange.py

```diff
@@ -149,6 +149,12 @@ def get_has_websocket(exchange_manager) -> bool:
     return exchange_manager.has_websocket
 
 
+def supports_websockets(exchange_name: str, tentacles_setup_config) -> bool:
+    exchange_manager = exchanges.ExchangeManager({}, exchange_name)
```

Comment:
I don't think it's a good idea to create ExchangeManager instances to check if a list of exchanges supports websocket

Reply:
The problem is that we need to go there to check a websocket https://github.com/Drakkar-Software/OctoBot-Trading/blob/master/octobot_trading/exchanges/implementations/ccxt_websocket_exchange.py#L28 (or we can reimplement a new method but that will duplicate some code)

Reply:
We should implement it differently